### ISSUE TITLE
Starting any stack results in an empty service

### ISSUE THREAD

**Issue by rossigee** [rancher/rancher#5534]
If I try to start a stack using a 'docker-compose.yml' and 'rancher-compose.yml', the stack is started but no services exist within it. The same happens if I select any application from the rancher default catalog.

I couldn't see anything out of the ordinary in the rancher-server logs that would explain it.
---

| Useful | Info |
| :-- | :-- |
| Versions | Rancher `v1.1.2` Cattle: `v0.165.8` UI: `v1.1.12` |
| Access | `openldap` `admin` |
| Orchestration | `Cattle` |
| Route | `environments.index` |

**Comment by tfiduccia** [comment]
Can you tell me which catalog you tried from default.  I can't currently reproduce this.

**Comment by rossigee** [comment]
I've tried many stacks - even simple ones - whether they are run from the catalog, or by manually running up a stack and specifying the docker-compose and rancher-compose content, the result is a stack with no services.

I suspect there's some kind of issue running docker-compose/rancher-compose under the hood, but I've not seen any related errors in the logs of the rancher-server container that would help me to narrow it down. And, I would expect it to fail completely, not leave me with an empty stack!?

I should note that this is only happening on *one* Rancher domain I administer. I also administer another two, which are running the same versions, but work just fine. This one *was* working fine before, but isn't any more. It might have stopped working around the time we moved the rancher-server to another ISP, but I can't see any issues with the networking (no firewall between them).

Any ideas what I should/could check to investigate this further?

**Comment by chrisdupuy** [comment]
@rossigee : exactly the same issue here. Impossible to create new stack manually from docker-compose or directly from catalog. The stack remains empty with "Activating (In Progress)" status. Old stack still running perfectly.

I run Rancher in HA with external database (Galera) : 
Component			Version
Rancher				v1.1.1
Cattle				v0.165.7
User Interface			v1.1.12
Rancher Compose		v0.8.6

**Comment by chrisdupuy** [comment]
Update : I can use rancher-compose perfectly to create stack. Only the UI is impacted.
Any idea ?

**Comment by sangeethah** [comment]
@chrisdupuy  , in your HA setup , can you check if both the instances of rancher-compose-executor in management stack in "System HA" environment  are in "Running" state ? If one of them is in "Stopped" state , that could be the reason for the behavior you are seeing.

**Comment by chrisdupuy** [comment]
@sangeethah, all rancher-compose-executor are in running state. But the state of the 'management' stack is also in Activating(in progress). I don"t know if one service is missing. Here are all my services : 


 Active	cattle  	Image: rancher/none	Service	6 Containers	 
 Active	go-machine-service  	Image: rancher/server:v1.1.2	Service	2 Containers	 
 Active	load-balancer  	To: websocket-proxy, cattle Ports: 80, 443, 81, 444	Load Balancer	3 Containers	 
 Active	load-balancer-swarm  	To: websocket-proxy-ssl Ports: 2376/tcp	Load Balancer	3 Containers	 
 Active	rancher-compose-executor  	Image: rancher/server:v1.1.2	Service	2 Containers	 
 Active	redis  	Image: rancher/none	Service	3 Containers	 
 Active	tunnel  	Image: rancher/none	Service	36 Containers	 
 Active	websocket-proxy  	Image: rancher/server:v1.1.2	Service	2 Containers	 
 Active	websocket-proxy-ssl  	Image: rancher/server:v1.1.2	Service	2 Containers	 
 Active	zookeeper  	Image: rancher/none	Service	3 Containers

Do you see something missing? 
It is a fresh install of HA cluster with nothing else than one dockerhub registry and a ssl certificate configuration.

**Comment by aemneina** [comment]
@rossigee is there anything in the logs (when viewed from the host) for the go-machine service or rancher compose executor?

@chrisdupuy what does the Admin->HA tab display? Sounds like your HA setup is mostly healthy but some minor hiccup where the management stack isnt showing fully activated. Do container/host stats appear?

**Comment by rossigee** [comment]
I get this repeating several times a minute in the rancher-server logs...

```
time="2016-09-07T16:01:03Z" level=fatal msg="Unable to start event router" error="Timed out waiting for transtion." 
time="2016-09-07T16:01:03Z" level=info msg="Starting rancher-compose-executor" version=v0.8.6 
time="2016-09-07T16:01:08Z" level=fatal msg="Exiting go-machine-service: Timed out waiting for transtion." 
time="2016-09-07T16:01:09Z" level=info msg="Setting log level" logLevel=info 
time="2016-09-07T16:01:09Z" level=info msg="Starting go-machine-service..." gitcommit=v0.31.7 
time="2016-09-07T16:01:09Z" level=info msg="Waiting for handler registration (1/2)" 
```

**Comment by aemneina** [comment]
hey @rossigee did you ever get this stabilized? the error you see usually indicates web socket proxy is hurt somehow... please reopen if youre still seeing this issue.

**Comment by rossigee** [comment]
IIRC, this resolved itself during one of the previous 1.1 upgrades.
